### ISSUE TITLE
Version '2.8.7' for 'arangodb' was not found 

### ISSUE THREAD

**Issue by kikohs** [arangodb/arangodb#1830]
Hello,
I am trying to install arangodb on Ubuntu 14.04 LTS.
I followed the instruction on the website.

My `arangodb.list` contains:

    deb https://www.arangodb.com/repositories/arangodb2/xUbuntu_14.04/ /

And I see the files online [here](https://www.arangodb.com/repositories/arangodb2/xUbuntu_14.04/)

Any hints?

**Comment by dothebart** [comment]
hm, the matching file can be found here:

https://www.arangodb.com/repositories/arangodb2/xUbuntu_14.04/amd64/arangodb_2.8.7_amd64.deb

as expected; do you get any more errors like certificate errors? 
Maybe it was a non-permanent connectivity issue inbetween our servers, can you re-check? In doubt downloading and dpkg -i'ing the file from the above link should also work.

**Comment by kikohs** [comment]
Still an issue.
That's all I've got:

```
sudo apt-get install arangodb=2.8.7
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Version '2.8.7' for 'arangodb' was not found
```

Dkpg install works though.

**Comment by dothebart** [comment]
Ok, I revalidated this. The Packages.gz files generated by SuSE OBS are only offer the latest version of a release series. So once 2.8.8 is released, 2.8.7 will not be available anymore. 

If one wants older versions in a release cycle, the wget & dpkg way is the way to go.
